docs/pages/setup/guides/joining-nodes-aws.mdx

```diff
@@ -53,14 +93,62 @@ Teleport will use to authenticate. You can provide the IAM credentials to
 teleport through a shared configuration file or environment variables, see
 https://aws.github.io/aws-sdk-go-v2/docs/configuring-sdk/#specifying-credentials
 for details.
+  </TabItem>
+</Tabs>
 
 ## Step 2/4. Create the AWS Node Joining Token
```

Comment:
```suggestion
## Step 2/4. Create the AWS Node Joining token
```
To make the heading sentence case (unless we're intending to capitalize "AWS Node Joining Token" everywhere?)